maths/special_numbers/triangular_numbers.py

```diff
@@ -0,0 +1,38 @@
+"""
+A triangular number or triangle number counts objects arranged in an
+equilateral triangle. This module provides a function to generate n'th
+triangular number.
+
+For more information about triangular numbers, refer to:
+https://en.wikipedia.org/wiki/Triangular_number
+"""
+
+
+def nth_triangular_number(position: int) -> int:
+    """
+    Generate the triangular number at the specified position.
+
+    Args:
+        position (int): The position of the triangular number to generate.
+
+    Returns:
+        int: The triangular number at the specified position.
+
+    Examples:
+    >>> nth_triangular_number(1)
+    1
+    >>> nth_triangular_number(3)
+    6
+    >>> nth_triangular_number(5)
+    15
+    """
+    if position < 1:
+        raise ValueError("param `position` must be greater than 0")
```

Comment:
1. The triangular numbers are also defined for n = 0 (see Wikipedia)
2. Could you add a doctest for this ValueError case?

Reply:
Okay